NethermindNode.Core/Helpers/NodeInfo.cs

```diff
@@ -216,4 +218,17 @@ public static bool VerifyLogsForUndesiredEntries(ref List<string> errors)
 
         return status;
     }
+
+    public static async Task<long> GetMergeBlockNumber()
+    {
+        var netType = await GetNetworkType(TestLoggerContext.Logger);
+        switch (netType)
+        {
+
+            case NetworkType.Sepolia: return 1000000;
+            case NetworkType.Hoodi: return 100000;
+            case NetworkType.Holesky: return 100000;
```

Comment:
For the sake of speed - I decided to use relatively small number. probably we could update it to be more realistic